### PR TITLE
CS enrollment

### PR DESCRIPTION
**Pull request by unglikteng**

### CODE REVIEW

File: _data/authors.yml [hunk 1]
```diff
@@ -181,7 +181,24 @@ nathan_smith:
...
+lik_teng_ung:
+    name: Lik Teng Ung 
+    title: Stack Developer
+    bio: Ung is a stack contributor at the Daily Bruin and a third-year statistics and computer science & linguistics student. He enjoyed boba, ramen and other things that are sugary.
+    email: lung@media.ucla.edu
+    mug: Yes
+keith_atienza:
+    name: Keith Atienza
+    title: Stack Developer
+    bio: Hello I am Keith
```
Comment (nathunsmitty) [inline _data/authors.yml]: hi keith

File: _posts/2018-11-13-cs-enrollment.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    authors:
+    - keith_atienza
+    - lik_teng_ung
+    - mattie_sanseverino
+    key_takeaways: 
+    - Applications and enrollment in the CS department has greatly increased over the years.
+    - The student to faculty ratio within HSSEAS and the CS department has grown since 2013.
+    - Since 2009, upper and lower division lectures have increased in size. 
+    featured_image:
+    url: research-funding/graph.png
```
Comment (nathunsmitty) [inline _posts/2018-11-13-cs-enrollment.md]: i think this should be a different image

File: _posts/2018-11-13-cs-enrollment.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+    - //cdn.plot.ly/plotly-latest.min.js
+    - //d3js.org/d3.v5.min.js
+    - //use.typekit.com/dkx5efl.js
+    - //unpkg.com/d3-simple-slider
+    - /js/posts/cs-enrollment/tiles.js
+    - /js/posts/cs-enrollment/jquery-3.3.1.min.js
+    - /js/posts/cs-enrollment/waffle.js
+---
+
+<br>
```
Comment (nathunsmitty) [inline _posts/2018-11-13-cs-enrollment.md]:
```suggestion
```
There shouldn't be a br here

File: _posts/2018-11-13-cs-enrollment.md [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+
+<br> 
+## CS Applications in context
+
+<div class='waffle'></div>
+
+
+
+## CS vs. Engineering Faculty 
+<br>
```
Comment (nathunsmitty) [inline _posts/2018-11-13-cs-enrollment.md]:
```suggestion
```

🔥

File: _posts/2018-11-13-cs-enrollment.md [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+<p>Below you can see how the amount of faculty within the Computer Science department has changed relative to the faculty in the overall School of Engineering. </p>
```
Comment (nathunsmitty) [inline _posts/2018-11-13-cs-enrollment.md]: don't use `p` tags in markdown. markdown handles that automatically

File: _posts/2018-11-13-cs-enrollment.md [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+<br> 
+
+<iframe align="center" width="85%" height="400" frameborder="0" scrolling="no" src="//plot.ly/~MattieSanseverino/5.embed?link=false&modebar=false&width=100%&height=50%"></iframe>
```
Comment (nathunsmitty) [inline _posts/2018-11-13-cs-enrollment.md]: what's going on here? we shouldn't be using iframes. I thought plot.ly was a library, similar to d3?

File: _posts/2018-11-13-cs-enrollment.md [hunk 7]
```diff
@@ -0,0 +1,105 @@
...
+## CS Applications in context
+
+<div class='waffle'></div>
+
+
+
+## CS vs. Engineering Faculty 
+<br>
+<p>Below you can see how the amount of faculty within the Computer Science department has changed relative to the faculty in the overall School of Engineering. </p>
+<br>
```
Comment (nathunsmitty) [inline _posts/2018-11-13-cs-enrollment.md]:
```suggestion
```

probably gonna not mark every p and br tag that needs to be removed now

File: _posts/2018-11-13-cs-enrollment.md [hunk 8]
```diff
@@ -0,0 +1,105 @@
...
+<p><i>FTE stands for Full-Time-Equivalent, and according to UCLA’s Workforce Methodology, FTE is calculated based on the number of hours per week that faculty members work. For example, an FTE of 0.5 is given for a person who works half-time (20 hours per week) while an FTE of 1.0 is given for a full-time employee. </i></p>
+
+
+<br>
+<p>While the total enrollment in the Henry Samueli School of Engineering rose 9.64% 2013 to 2016, from 3,160 to 3,480 enrolled undergraduate students, the total rise in faculty FTE within the School of Engineering rose 1.13%. Thus, the student to faculty FTE ratio has changed from 3.65:1.00 to 4.00:1.00. </p>
+<br>
+<p>Since the enrollment in the Computer Science Department has grown dramatically, the percent of CS/CSE majors within HSSEAS rising from 13.7% in 2013 to 21.0% in 2016, one would expect the growth of FTE within the Computer Science Department to grow more dramatically than that of the HSSEAS, as well as reflect the growth in enrolled students. However, the number of enrolled students in the department increased significantly compared to the growth in FTE, with the student to FTE ratio rising from ___ in 2013 to ___ in 2016. </p>
```
Comment (nathunsmitty) [inline _posts/2018-11-13-cs-enrollment.md]: ___ -> whatever that should be

File: _posts/2018-11-13-cs-enrollment.md [hunk 9]
```diff
@@ -0,0 +1,105 @@
...
+
+
+<br>
+<br>
+## Heatmap for CS Lecture and Section Size
+As the enrollment for CS and CSE majors (including freshmen and transfer) increases by 66% from 2000 to 2018, we also found that the enrollment number for CS classes has more than doubled from 2009 to 2017. Despite the surging demand, the number of CS classes offered has barely increased from 2009 to 2017. Some rare exceptions are CS 35L and CS M152A, which are classes that are taught by TA. 
+<br><br>
+As there are more students at UCLA, he average lecture size for CS classes increased from 36 in 2009 to 80 in 2017. You can use the interactive chart to examine how the lecture size for each CS course has changed from 2009-2017. Interestingly, we found out that more sections have been added to each lectures to accommodate more students and have kept the size of the sections pretty consistent at 30 to 50 students from 2009 to 2017.
+
+<div id="container">
```
Comment (nathunsmitty) [inline _posts/2018-11-13-cs-enrollment.md]: what's going on here?

File: css/posts/cs-course-enrollment/reset.css [hunk 10]
```diff
@@ -0,0 +1,46 @@
+html, body, div, span, applet, object, iframe,
```
Comment (nathunsmitty) [inline css/posts/cs-course-enrollment/reset.css]: why are we touching things like html and body?

File: css/posts/cs-course-enrollment/reset.css [hunk 11]
```diff
@@ -0,0 +1,46 @@
...
+	text-align: left;
+	font-weight: normal;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: "";
+}
+blockquote, q {
+	quotes: "" "";
+}
```
Comment (nathunsmitty) [inline css/posts/cs-course-enrollment/reset.css]: add a newline

File: css/posts/cs-course-enrollment/tiles.css [hunk 12]
```diff
@@ -0,0 +1,365 @@
...
+	fill: #ddd;
+}
+
+h2.title {
+	/* margin-top: -2px; */
+	font-size: 24px;
+	line-height: 28px;
+	margin-bottom: 5px;
+	width: 960px;
+	/* margin-left: -50px; */
```
Comment (nathunsmitty) [inline css/posts/cs-course-enrollment/tiles.css]: remove all comments before the final version

File: css/posts/cs-enrollment/app.css [hunk 13]
```diff
@@ -0,0 +1,46 @@
...
+
+
+  -webkit-appearance: none;
+  -moz-appearance:    none;
+  appearance:         none;
+
+  justify-content: center;
+  align-items: center;
+
+  font-size: 90%; */
```
Comment (nathunsmitty) [inline css/posts/cs-enrollment/app.css]:
```suggestion
  font-size: 90%;
```

File: datasets/cs-enrollment/cs_fte.csv [hunk 14]
```diff
@@ -0,0 +1,7 @@
+﻿Year,FTE
+2011,31
+2012,30.67
+2013,30.67
+2014,30.67
+2015,34.67
+2016,39.17
```
Comment (nathunsmitty) [inline datasets/cs-enrollment/cs_fte.csv]: add a newline to the ends of these files

File: js/posts/cs-enrollment/bar_graph.js [hunk 15]
```diff
@@ -0,0 +1,54 @@
+var engineering_data;
+$.getCSV('/datasets/cs-enrollment/eng_fte_totals.csv', function(data) {
+  engineering_data = data;
+});
+
+function tooltip_contents(d, defaultTitleFormat, defaultValueFormat, color) {
```
Comment (nathunsmitty) [inline js/posts/cs-enrollment/bar_graph.js]: are we using `tooltip_contents` anywhere?

File: js/posts/cs-enrollment/bar_graph.js [hunk 16]
```diff
@@ -0,0 +1,54 @@
...
+  var $$ = this, config = $$.config, CLASS = $$.CLASS,
+      titleFormat = config.tooltip_format_title || defaultTitleFormat,
+      nameFormat = config.tooltip_format_name || function (name) {return name; },
+      valueFormat = config.tooltop_format_value || defaultValueFormat,
+      text, i, title, value, name, bgcolor;
+} //do we need the for loop stuff?
+
+var titles = ['http://c3js.org/samples/tooltip_format.html', 'b'];
```
Comment (nathunsmitty) [inline js/posts/cs-enrollment/bar_graph.js]: prefer let or const to var

File: js/posts/cs-enrollment/bar_graph.js [hunk 17]
```diff
@@ -0,0 +1,54 @@
+var engineering_data;
+$.getCSV('/datasets/cs-enrollment/eng_fte_totals.csv', function(data) {
+  engineering_data = data;
+});
+
+function tooltip_contents(d, defaultTitleFormat, defaultValueFormat, color) {
+  var $$ = this, config = $$.config, CLASS = $$.CLASS,
+      titleFormat = config.tooltip_format_title || defaultTitleFormat,
+      nameFormat = config.tooltip_format_name || function (name) {return name; },
```
Comment (nathunsmitty) [inline js/posts/cs-enrollment/bar_graph.js]:
```suggestion
      nameFormat = config.tooltip_format_name || name => name,
```

File: js/posts/cs-enrollment/bar_graph.js [hunk 18]
```diff
@@ -0,0 +1,54 @@
...
+      valueFormat = config.tooltop_format_value || defaultValueFormat,
+      text, i, title, value, name, bgcolor;
+} //do we need the for loop stuff?
+
+var titles = ['http://c3js.org/samples/tooltip_format.html', 'b'];
+var titles2 = ['vva', 'g'];
+
+var fte_comparison_bar_chart = c3.generate({
```
Comment (nathunsmitty) [inline js/posts/cs-enrollment/bar_graph.js]: do we use `fte_comparison_bar_chart` anywhere?

File: js/posts/cs-enrollment/ratio-linechart.js [hunk 19]
```diff
@@ -0,0 +1,9 @@
+TESTER = document.getElementById('tester');
+
+Plotly.plot( TESTER, [{
+    x: [1, 2, 3, 4, 5],
+    y: [1, 2, 4, 8, 16] }], {
+    margin: { t: 0 } } );
+
+/* Current Plotly.js version */
+console.log( Plotly.BUILD );
```
Comment (nathunsmitty) [inline js/posts/cs-enrollment/ratio-linechart.js]: remove this before release

File: js/posts/cs-enrollment/ratio-linechart.js [hunk 20]
```diff
@@ -0,0 +1,9 @@
+TESTER = document.getElementById('tester');
```
Comment (nathunsmitty) [inline js/posts/cs-enrollment/ratio-linechart.js]: declare yo variable before you assign it

File: _posts/2018-11-13-cs-enrollment.md (hunk 6 above)
Comment (mattiesansev) [reply _posts/2018-11-13-cs-enrollment.md]: plot.ly doesn't have a good way to size the graph correctly and it kept turning up too large -- it was recommended to use iframe to fix this problem

**Pull request merged**